src/auth/guards/jwt-cookie/jwt-cookie-strategy.service.ts

```diff
@@ -32,7 +32,9 @@ export class JwtCookieStrategy extends PassportStrategy(
       this.couchdbService.get('app', data['username']),
     ).catch(() => {});
 
+    // TODO
```

Comment:
What is missing here?

Reply:
```suggestion
```

:see_no_evil:   I just forgot to remove the TODO comment that I had added after screening the code